packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/entityFeatures.ts

```diff
@@ -191,6 +196,216 @@ function cacheGetNeighborEntityElement(
     return element;
 }
 
+/**
+ * Content edit feature to move the cursor from Delimiter After Entity to the delimiter Before
+ */
+const MoveBeforeDelimiterFeature: BuildInEditFeature<PluginKeyboardEvent> = {
+    keys: [Keys.LEFT],
```

Comment:
Do we need special handling for RTL?

Reply:
Ah I see, yes we need to do that.

Reply:
Should be fixed in latest commit
![Delimiter2](https://user-images.githubusercontent.com/8291124/221705099-90dfe352-86d6-4652-9af7-97018aaa02fd.gif)